src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryControllerTests.kt

```diff
@@ -1,153 +1,175 @@
+        .andExpect(jsonPath("$.games", equalTo(library.games)))
     }
 
-    @Nested
-    @DisplayName("Tests for createLibrary")
-    inner class CreateLibrary {
-      // creates library successfully
-      @Test
-      fun shouldSuccessfullyCreateLibraryWithNoGamesPassedIn() {
-        val library = Library(id2, "Backlog", ArrayList())
-        given(libraryService.create(any())).willReturn(library)
-        val libraryRequest = LibraryRequest("Backlog", null)
-        requestBuilder.runPostRequest(endpoint, requestToString(libraryRequest))
-          .andExpect(status().isCreated)
-          .andExpect(jsonPath("$.name", equalTo(library.name)))
-          .andExpect(jsonPath("$.games", equalTo(library.games)))
-      }
+    @Test
+    fun shouldSuccessfullyCreateLibraryWithGamesPassedIn() {
+      val library = Library(id2, "Backlog", ArrayList())
+      library.games.add(id1)
+      given(libraryService.create(any())).willReturn(library)
+      val libraryRequest = LibraryRequest("Backlog", arrayListOf(id1))
+      requestBuilder.runPostRequest(endpoint, requestToString(libraryRequest))
+        .andExpect(status().isCreated)
+        .andExpect(jsonPath("$.name", equalTo(library.name)))
+        .andExpect(jsonPath("$.games[0]", equalTo(id1)))
+    }
+  }
 
-      @Test
-      fun shouldSuccessfullyCreateLibraryWithGamesPassedIn() {
-        val library = Library(id2, "Backlog", ArrayList())
+  @Nested
+  @DisplayName("Tests for addToLibrary")
+  // TODO: update tests when master library functionality is complete
+  inner class AddToLibrary {
+    @Test
+    fun shouldSuccessfullyAddToLibrary() {
+      val library = Library(id2, "Backlog", ArrayList())
+      given(libraryService.addToLibrary(any(), any())).will {
         library.games.add(id1)
-        given(libraryService.create(any())).willReturn(library)
-        val libraryRequest = LibraryRequest("Backlog", arrayListOf(id1))
-        requestBuilder.runPostRequest(endpoint, requestToString(libraryRequest))
-          .andExpect(status().isCreated)
-          .andExpect(jsonPath("$.name", equalTo(library.name)))
-          .andExpect(jsonPath("$.games[0]", equalTo(id1)))
       }
+      val map = HashMap<String, String>()
+      map["gameId"] = id1
+      requestBuilder.runPutRequest("$endpoint/$id2/addToLibrary", Gson().toJson(map))
+        .andExpect(status().isOk)
+      Assertions.assertTrue(library.games.contains(id1))
     }
+  }
 
-    @Nested
-    @DisplayName("Tests for addToLibrary")
-    // TODO: write these tests when master library functionality is complete
-    inner class AddToLibrary {
-      // adds to library successfully
-      // adds duplicate
-    }
-
-    @Nested
-    @DisplayName("Tests for deleteLibrary")
-    inner class DeleteLibrary {
-      // successfully deletes
+  @Nested
+  @DisplayName("Tests for deleteLibrary")
+  inner class DeleteLibrary {
+    @Test
+    fun shouldSuccessfullyDeleteFromLibrary() {
```

Comment:
```suggestion
    fun shouldSuccessfullyDeleteLibrary() {
```
We're deleting the library, not a game from it, right?

Reply:
yep thanks :)